package.json

```
@@ -14,12 +14,12 @@
     "concat-stream": "^2.0.0",
     "debug": "^4.0.0",
     "extend": "^3.0.0",
-    "glob": "^10.0.0",
+    "glob": "^11.0.0",
```

[`glob@11`](https://github.com/isaacs/node-glob/blob/main/changelog.md#110) removed support for Node.js < 20. We still support Node.js 16.

```suggestion
    "glob": "^10.0.0",
```

---

Or maybe

```suggestion
    "glob": "^10.0.0 || ^11.0.0",
```